src/GitHub.App/Api/ApiClient.cs

```diff
@@ -62,44 +63,30 @@ public IObservable<Gist> CreateGist(NewGist newGist)
             return gitHubClient.Gist.Create(newGist);
         }
 
-        public IObservable<User> GetUser()
+        public IObservable<UserAndScopes> GetUser()
         {
-            return gitHubClient.User.Current();
+            return GetUserInternal().ToObservable();
         }
 
-        public IObservable<string[]> GetScopes()
+        async Task<UserAndScopes> GetUserInternal()
         {
-            return GetScopesInternal().ToObservable();
-        }
-
-        async Task<string[]> GetScopesInternal()
-        {
-            var connection = gitHubClient.Connection;
+            var response = await gitHubClient.Connection.Get<User>(
+                new Uri("user", UriKind.Relative), null, null);
```

Comment:
This Uri should be a static readonly field (`userEndpoint`). Also, call `ConfigureAwait(false)` on this, it doesn't have to return to the caller thread (same pattern that the original Octokit call has)